app/lib/utils.js

```diff
@@ -6,7 +6,13 @@ function deepClone(obj) {
   return JSON.parse(JSON.stringify(obj));
 }
 
+function removeDuplicates(myArr, prop) {
```

Comment:
Nice, clean implementation.

Disappointing that javascript's Set data structure isn't usable for this due to lack of ability to use a custom equality  function.